lib/connect.js

```diff
@@ -303,6 +304,9 @@ function connect(token, options) {
  *   for the {@link LocalParticipant} to {@link NetworkQualityVerbosity}<code style="padding:0 0">#minimal</code>
  *   and the {@link NetworkQualityVerbosity} for {@link RemoteParticipant}s to
  *   {@link NetworkQualityVerbosity}<code style="padding:0 0">#none</code>.
+ * @property {?string} [region='gll'] - Preferred region. This region will be used
```

Comment:
@makarandp0 Two things here:
1. Can you convert `{?string} to `{string}`? A "?" implies that the argument is nullable.

2. Can we make the description closer to the spec:
```
- Preferred region; By default, the client will connect to the nearest signaling server
determined by latency based routing. Setting a value other than <code style="padding:0 0">gll</code>
bypasses routing and guarantees that signaling traffic will be terminated in the region that you prefer;
If you are connecting to a Group Room created with the <code style="padding:0 0">gll</code>
Media Region (either Ad-Hoc or via the REST API), then the Room's Media Region will be selected
based upon your Client's region
```

Reply:
Thanks. makes sense for both.